### ISSUE TITLE
New Task: Make Reused PTT Labeling Consistent

### ISSUE THREAD

**Issue by iwensu0313** [ioos/marine_life_data_network#111]
### Who is requesting this?

ATN DAC steering group, funders, etc

### What is being requested?

PTT labels are re-used and this is a known and expected thing in the ATN data provider community. In the ATN Portal Project Overview tab, the re-used PTT labels are marked with an extra suffix, e.g. `Franciscana dolphin (111838)[1]`, `Franciscana dolphin (111838)[2]`. The request is to remove the extra suffix to show that these two deployments are re-using same PTT label `[1]` `[2]`. However, the color differentiation of unique deployments needs to be preserved even if using the same PTT label.

### What is the current status quo (i.e., what happens if this does not get done)?

Inconsistent labeling between the PTT labels on the ATN Portal Project Overview tab and the ATN Portal Deployments tab

### Provide a description or any other important information.

NOTE: I need to touch base with our developers to fill out more details

**Comment by laurabrenskelle** [comment]
@mmckinzie Why is it important to maintain the fact that deployments had the same PTT label in the archived data?

@iwensu0313 Would it be easier to transform these IDs into something like 111838-1, 111838-2 (or some other minor change that allows them to be unique enough for the portal to function but might be recognizable to a human that they are using the same PTT number)?

**Comment by mmckinzie** [comment]
@laurabrenskelle This request is related to track visualization in the portal and the scheme used for labeling individual deployments. We established an agreed upon scheme years ago, i.e., species (ptt), even for cases where ptt IDs are reused, as reusing ptt IDs is standard practice and not unexpected. They are also clearly listed as individual deployments under the deployments tab, with clear data ranges, and tracks are visualized using a different-colored track line. 

However, fairly recently, the labeling schema of select deployments changed from species (ptt) to species (ptt)[0], species (ptt)[1], species (ptt)[2], etc., but only in select places within the portal (so the same deployment may be labeled as just species (ptt) in one place but species (ptt)[2] in another) and this change only applied to newly visualized data. 

The point was to revert back to a system where labels are the same across all deployments in all places, portal-wide, and follow the original, preferred schema. Or, I suppose, we could change to a different labeling schema but also ensure that the change is presented consistently across places and for all deployments. However, the latter is not our preference and would require significantly more effort/time/cost, as many deployments would have to be manually reprocessed.

**Comment by MathewBiddle** [comment]
@mmckinzie @laurabrenskelle Does this impact the NCEI pipeline?

Have we learned something new in the last few months that would change the purpose of this issue? 

The issue was blocked due to a gap in funding. Is this something we should evaluate to pick up again?

**Comment by mmckinzie** [comment]
No, this does not impact the NCEI pipeline. The ADR references unique deployment IDs to generate data files, and by combining species, ptt ID and deployment start and end dates in the file-naming schema, it will always create a unique file name for the netCDF files we serve to NCEI.

This issue is strictly related to how deployments are labeled in the data portal. This is also not a visualization issue. Deployments with reused ptt IDs are shown as distinct deployments, and they appear on the map in different colors, as expected. That must be preserved. 

The ask is to remove the suffix (e.g., [1], [2]) in cases where a ptt ID is reused and resume labeling deployment by species (ptt) only, not species (ptt) [0], species (ptt) [1], species (ptt) [2]...At some point, a change was made that started adding the suffix, so older projects don't have it, whereas newer ones do. 

This is a low-priority issue, but I'd like to fix it, if possible, to ensure consistency in deployment labeling across projects in the portal.
